source/1_engine/rex_engine/src/filesystem/path.cpp

```diff
@@ -81,6 +81,16 @@ namespace rex
       return g_seperation_char;
     }
 
+    // removes leading and trailing quotes from a path
+    rsl::string_view remove_quotes(rsl::string_view path)
+    {
+      if (path.starts_with("\"") && path.ends_with("\""))
+      {
+        path = path.substr(1, path.length() - 2);
+      }
+
+      return path;
+    }
```

Comment:
true, but it's in the path namespace, where else would you expect it to remove quotes?

Reply:
Well, I could have made a typo and have two quotes at the start and one at the end it will not capture that. 
Or I could have quotes within my path, I know this is not a valid path, but people make mistakes.

Just getting ahead of problems that might occur.

Reply:
if you want to remove quotes in the middle, you can't return a string view, it'll have to get reverted to a string.

Reply:
regarding a mis match in the number of quotes at the start and the end, yes that's fixable, but that'd change the behavior of this function. I've had that before and had to change it because it caused an issue, although I don't remember what caused it

Reply:
But the most important part is that I would make it clear that it will only remove if start AND end have quotes. if you only have quotes at the start it won't do anything.